apps/storybook/src/Table.stories.tsx

```diff
@@ -3130,80 +2983,73 @@ export const ColumnManager: Story<Partial<TableProps>> = (args) => {
   const columns = useMemo(
     (): Column<TableStoryDataType>[] => [
       {
-        Header: 'Table',
-        columns: [
-          {
-            id: 'index',
-            Header: '#',
-            accessor: 'index',
-            disableToggleVisibility: true,
-          },
-          {
-            id: 'name',
-            Header: 'Name',
-            accessor: 'name',
-          },
-          {
-            id: 'description',
-            Header: 'Description',
-            accessor: 'description',
-            fieldType: 'text',
-          },
-          {
-            id: 'id',
-            Header: 'ID',
-            accessor: 'id',
-          },
-          {
-            id: 'startDate',
-            Header: 'Start date',
-            accessor: 'startDate',
-            Cell: (props: CellProps<TableStoryDataType>) => {
-              return (
-                <>{props.row.original.startDate.toLocaleDateString('en-US')}</>
-              );
-            },
-          },
-          {
-            id: 'endDate',
-            Header: 'End date',
-            accessor: 'endDate',
-            Cell: (props: CellProps<TableStoryDataType>) => {
-              return (
-                <>{props.row.original.endDate.toLocaleDateString('en-US')}</>
-              );
-            },
-          },
-          {
-            id: 'Price',
-            Header: 'Price',
-            accessor: 'price',
-          },
-          {
-            id: 'Color',
-            Header: 'Color',
-            accessor: 'color',
-          },
-          {
-            id: '# in stock',
-            Header: '# in stock',
-            accessor: 'stock',
-          },
-          {
-            id: 'Rating',
-            Header: 'Rating',
-            accessor: 'rating',
-          },
-          {
-            id: 'Location',
-            Header: 'Location',
-            accessor: 'location',
-          },
-          ActionColumn({
-            columnManager: true,
-          }),
-        ],
+        id: 'index',
+        Header: '#',
+        accessor: 'index',
+        disableToggleVisibility: true,
+      },
+      {
+        id: 'name',
+        Header: 'Name',
+        accessor: 'name',
+      },
+      {
+        id: 'description',
+        Header: 'Description',
+        accessor: 'description',
+        fieldType: 'text',
+      },
+      {
+        id: 'id',
+        Header: 'ID',
+        accessor: 'id',
+      },
+      {
+        id: 'startDate',
+        Header: 'Start date',
+        accessor: 'startDate',
+        Cell: (props: CellProps<TableStoryDataType>) => {
+          return (
+            <>{props.row.original.startDate.toLocaleDateString('en-US')}</>
+          );
+        },
+      },
+      {
+        id: 'endDate',
+        Header: 'End date',
+        accessor: 'endDate',
+        Cell: (props: CellProps<TableStoryDataType>) => {
+          return <>{props.row.original.endDate.toLocaleDateString('en-US')}</>;
```

Comment:
This return statement no longer has () around it. Is that okay?

Reply:
I believe this is some `prettier` print issue

Reply:
ah ok